src/components/insync/data-table/index.tsx

```diff
@@ -123,12 +125,20 @@ const DataTable: FunctionComponent<DataTableProps> = ({
 	const showRows = !loading && sortedData.length > 0;
 	const otherElement = <div className="flex justify-center md:px-15 md:pb-5 mt-5">{loading ? loader : noData}</div>;
 
-	if (isMobileView) {
+	if ((screenWidth && screenWidth <= 830) || isMobileView) {
```

Comment:
I think the 830 should come from a prop and `screenWidth` you can just get using the hook

Reply:
updated this